amgut/lib/data_access/redcap.py

```diff
@@ -0,0 +1,159 @@
+from urllib import urlencode
+from json import loads
+
+from tornado.web import HTTPError
+from tornado.httpclient import HTTPClient
+
+from amgut.lib.config_manager import AMGUT_CONFIG
+from amgut.lib.data_access.sql_connection import TRN
+
+
+# Explanation of redcap and interactions with it:
+# Redcap has three layers of IDs: record_id, survey_id, and event_id.
+
+# record_id is the id given to a single person in the redcap system, analagous
+# to the host_subject_id.
+
+# survey_id is the name of the survey the person will be taking.
+
+# event_id is the id of the single time a person has taken the survey. It
+# counts up from 1 for each person in each survey. These must be manually
+# created for each survey before the survey is used, so we don't want to waste
```

Comment:
Is the following accurate?

"An event_id is the ID of a single time a person has taken a survey. It corresponds to a single instance of a survey for a given survey type and user. Within REDCap, it is necessary to create these "instance" IDs in advance, and are static artifacts of the system. These IDs can be used to distinguish separate surveys that an individual has taken (e.g., multiple time points)."